Sources/Expression.swift

```diff
@@ -40,9 +47,13 @@ final class VariableExpression: Expression, CustomStringConvertible {
   var description: String {
     return "(variable: \(variable))"
   }
-
+  
+  func resolve(_ context: Context) throws -> Any? {
+    return try variable.resolve(context)
+  }
+  
   /// Resolves a variable in the given context as boolean
-  func resolve(context: Context, variable: Resolvable) throws -> Bool {
+  func evaluate(context: Context) throws -> Bool {
```

Comment:
Weird, I think I've seen this change a few times in other PRs? 😕😆

Reply:
maybe, some features required similar changes